sw/airborne/boards/bebop/mt9f002.c

```diff
@@ -65,7 +65,14 @@ struct video_config_t front_camera = {
   .buf_cnt = 5,
   .filters = VIDEO_FILTER_ISP,
   .cv_listener = NULL,
-  .fps = MT9F002_TARGET_FPS
+  .fps = MT9F002_TARGET_FPS,
+  .camera_intrinsics = {
+    .focal_x = MT9F002_FOCAL_X;
+    .focal_y = MT9F002_FOCAL_Y;
+    .center_x = MT9F002_CENTER_X;
+    .center_y = MT9F002_CENTER_Y;
+    .Dhane_k = MT9F002_DHANE_K;
+  }
```

Comment:
You should define this for the bottom cam as well.

Reply:
At the moment the bottom cam is derotated in the old way. It can be added later in my eyes.